templates/python/custom-copilot-assistant-new/teamsapp.local.yml.tpl

```diff
@@ -8,7 +8,7 @@ provision:
   - uses: teamsApp/create
     with:
       # Teams app name
-      name: {{appName}}-${{TEAMSFX_ENV}}
+      name: {{appName}}${{APP_NAME_SUFFIX}}
```

Comment:
What will happen if `.env.xxx` files do not contain this environment variable? Do we need to update the env files to include this new environment variable?

Reply:
all .env.XXX files include APP_NAME_SUFFIX if the related .yml files use this variable

Reply:
Per frank, all .env files contain the variable. Resolving.